apps/calendar/js/notification.js

```diff
@@ -21,19 +21,21 @@ exports.sendNotification = co.wrap(function *(title, body, url, data) {
   }
 
   var icon = `${NotificationHelper.getIconURI(app)}?${url}`;
-  var notification = new Notification(title, {
-    body: body,
+  return NotificationHelper.send(titleL10n, {
```

Comment:
Since we're inside of a co wrapped generator function, we can use `yield` to wait for a promise to resolve instead of `.then()`